modules/installation-configuration-parameters.adoc

```diff
@@ -3501,6 +3501,13 @@ For more information on usage, see "Configuring a failure domain" in "Installing
 |The password for the Prism Central user name.
 |String
 
+|platform:
+  nutanix:
+    prismCentral:
+      preloadedOSImageName:
+|Instead of creating and uploading a new {op-system} image object for each {product-title} cluster, this parameter uses the named, preloaded {op-system} image object from the private cloud or the public cloud.
```

Comment:
tiny nit while I'm here - "creating" implies "new" so you can omit that description :)
```suggestion
|Instead of creating and uploading an {op-system} image object for each {product-title} cluster, this parameter uses the named, preloaded {op-system} image object from the private cloud or the public cloud.
```